tests/contrib/django_util/__init__.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
`oauth2client` is a library only warranted for use when imported by other Python code. No shebang lines, please.